docs/zq/functions/len.md

```diff
@@ -0,0 +1,43 @@
+### Function
+
+&emsp; **len** &mdash; the type-dependent length of a value
```

Comment:
```suggestion
&emsp; **len** &mdash; type-dependent length of a value
```

Reply:
This voice is not parallel with the other titles in reference.md